src/MultivariateStats.jl

```diff
@@ -1,11 +1,11 @@
 module MultivariateStats
     using Compat
     using StatsBase
-    using ArrayViews
 
     import Base: length, size, show, dump, sumabs2
     import Base.LinAlg: Cholesky
     import StatsBase: fit, predict
+    using  Compat: view
```

Comment:
You're `using Compat` twice; could make the latter an `import` though I don't think it really matters